.github/workflows/build.yml

```diff
@@ -26,9 +26,13 @@ jobs:
         with:
           submodules: recursive
 
+      - name: Check Cargo build
+        run: |
+          cargo test
+
       - name: Install dependencies
         run: |
-          cargo install pdl-compiler --version 0.2.2
+          cargo install pdl-compiler --version 0.2.3
```

Comment:
I'm assuming it's desirable for pdl-compiler to be up to date -- if you'd rather stick with 0.2.2 that's fine too of course